libs/wallet/src/connectors/rest-connector.ts

```diff
@@ -48,26 +85,22 @@ export class RestConnector implements VegaConnector {
     }
   ) {
     try {
-      const service = new DefaultApi(
-        createConfiguration({
-          baseServer: new ServerConfiguration<Record<string, never>>(url, {}),
-        })
-      );
+      this.url = url;
 
-      const res = await service.authTokenPost(params);
+      const res = await this.request('auth/token', {
```

Comment:
I dislike the number of magic strings for endpoints in here. could we store these in a map?

Reply:
Changed to enum